powermon/commands/result.py

```diff
@@ -1,40 +1,48 @@
 from enum import Enum, auto
 import logging
 
-from powermon.commands.response import Response
-from powermon.commands.response_definition import ResponseDefinition
+from powermon.commands.reading import Reading
+from powermon.commands.parameter import Parameter
+from powermon.commands.reading_definition import ReadingDefinition
 from powermon.dto.resultDTO import ResultDTO
 
 log = logging.getLogger("result")
 
 class ResultType(Enum):
     DEFAULT = auto()
     ACK = auto()
+    COMMAND = auto()
```

Comment:
not another command! :-P
a variation on ACK might be less confusing?

Reply:
think we need to define the options
1, ack type (success / fail of command)  ACK, COMMAND
2, a series of values, the place (1st, 2nd etc) in relation to overall defines what the value relates to, INDEXED, DEFAULT
3, a long 'string', where the position (eg characters 3-5) defines that the value relates to POSITIONAL??
4, a series of values that are all related to a single answer MULTIVALUED
5, key: value data (multiple per 'response') where the key defines what the value relates to  ??
6, multiframe (cant remember how this works off hand) ??
7, single value, SINGLE

Reply:
DEFAULT should go now, was a temp while the protocols were migrated to new approach in mppsolar code

Reply:
Yeah, we have enough commands, I'll rename it.

I will add more ResultTypes when I start converting the protocols. Do you like the pattern enough that I should start applying it to the whole protocol? It will take a little while.

Reply:
yeah, i think the pattern is great - better to have more types than lots of complexity in decode
I'm keen to merge as soon as possible (even unfinished) so I can make some changes without running into merge problems - are ok to merge soon?
Also have you seen the port PR (just some tidy atm, but Im wanting to play with BLE)??

Reply:
If you are okay with it being broken for a bit you can merge anytime. We can work together on fixing it up.